pkgs/by-name/pl/plex-desktop/update.sh

```diff
@@ -73,6 +73,7 @@ sed --regexp-extended \
 # try to build the updated version
 #
 
+export NIXPKGS_ALLOW_UNFREE=1
```

Comment:
I'm not sure if this is the best way to do this. I mean realistically this isn't a _big_ deal because its just a nixpkgs script

Reply:
With `export` specifically or allowing unfree packages this way in general? `env NIXPKGS_ALLOW_UNFREE=1 nix-build -A plex-desktop` probably isn't an improvement.

I'd have to look into doing this another way. I'm not sure what the straightforward answer is other than using the environment variable and using `nix-build` or `nix build --impure` in a shell script.

Reply:
honestly its probably fine. its just one less maintainer extra thing to do